### PR TITLE
Amend `next_week` documentation

### PR DESCRIPTION
**Pull request by iamvery**
I added examples of the `next_week` method for `Date` and `Time`.

### CODE REVIEW

**Comment by iamvery** [comment]
Sorry I failed to [skip the build](http://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#contributing-to-the-rails-documentation) on the commit :(

**Comment by senny** [comment]
@iamvery I don't think this addition is necessary. The very beginning of the documentation says:

```ruby
# Returns a new date/time representing the given day in the next week.
# The +given_day_in_next_week+ defaults to the beginning of the week
```

This already stats that you can pass the day you want to set it to. To clarify we can add an example below the text.

**Comment by iamvery** [comment]
Thanks for the feedback @senny. At the time, I didn't realize that `beginning_of_week` was actually a symbol itself. For some reason I had assumed it was a date or time value.

I updated the branch with examples instead of the new sentence. Is this more clear?

File: activesupport/lib/active_support/core_ext/date_and_time/calculations.rb [hunk 1]
```diff
@@ -128,6 +128,17 @@ def beginning_of_year
     # The +given_day_in_next_week+ defaults to the beginning of the week
     # which is determined by +Date.beginning_of_week+ or +config.beginning_of_week+
     # when set. +DateTime+ objects have their time set to 0:00 unless +same_time+ is true.
+    #
+    # ==== Examples
+    #   today = Date.today
+    #   # => Thu, 07 May 2015
+    #   today.next_week(:friday)
```
Comment (senny) [inline activesupport/lib/active_support/core_ext/date_and_time/calculations.rb]: Let's not pass the + given_day_in_next_week+ in the first example to showcase the default value.

File: activesupport/lib/active_support/core_ext/date_and_time/calculations.rb [hunk 2]
```diff
@@ -128,6 +128,17 @@ def beginning_of_year
     # The +given_day_in_next_week+ defaults to the beginning of the week
     # which is determined by +Date.beginning_of_week+ or +config.beginning_of_week+
     # when set. +DateTime+ objects have their time set to 0:00 unless +same_time+ is true.
+    #
+    # ==== Examples
+    #   today = Date.today
```
Comment (senny) [inline activesupport/lib/active_support/core_ext/date_and_time/calculations.rb]: we should be able to compress the example by putting the output on the same line:

```ruby
today = Date.today # => Thu, 07 May 2015
today.next_week    # => XYZ
```

**Comment by iamvery** [comment]
Updated!

**Comment by senny** [comment]
@iamvery thank you :yellow_heart: I merged a slightly modified version to match the examples to the prose (https://github.com/rails/rails/commit/64ed04ee78bd8d08bd91e997a0b93172be9d9f35).

**Comment by iamvery** [comment]
:sparkles: thank you for the quick, efficient help!

**Pull request merged**